swift/example_code/cognito-identity/CognitoIdentityDemo/README.md

```diff
@@ -0,0 +1,25 @@
+# CognitoIdentityDemo
+
+In this, the first example introduced in the _AWS SDK for Swift Developer's Guide_, we create a simple command-line program that uses your default AWS account, as configured using the AWS CLI or otherwise, to simply create a new Cognito Identity Pool, or to return the ID of the existing one if one's already present.
```

Comment:
Is the fact that this is the first example relevant here? Will it be relevant down the road when there are many more examples?